setup.py

```diff
@@ -23,8 +23,8 @@
       description=("A Python library for use with Dell EMC's Unisphere for "
                    "PowerMax REST API."),
       license='Apache 2.0',
-      packages=['PyU4V', 'PyU4V.utils'],
-      install_requires=['requests', 'six', 'urllib3'],
+      packages=['PyU4V', 'PyU4V.utils', 'PyU4V.tools.openstack'],
```

Comment:
Still scratching head over this one and how to best approach it, it came from this customer issue: https://github.com/MichaelMcAleer/PyU4V/issues/11 

I'm not sold on the prospect of having to list out all packagaes, sub-packaged, sub-subpackages etc.  Had a quick look online, we may be able to switch to `packages=setuptools.find_packages()` as directed in https://packaging.python.org/tutorials/packaging-projects/ under `'Creating setup.py'` and again here https://packaging.python.org/guides/distributing-packages-using-setuptools/#packages. 

Taking this approach and changing may save us from having to hard-code any packages, sub-packages, sub-subpackages etc moving forward.  One to discuss next time we meet.

Reply:
Absolutely.  I can give your suggestion a quick test.